bin/plugin/lib/git.js

```diff
@@ -18,7 +18,10 @@ const { getRandomTemporaryPath } = require( './utils' );
 async function clone( repositoryUrl ) {
 	const gitWorkingDirectoryPath = getRandomTemporaryPath();
 	const simpleGit = SimpleGit();
-	await simpleGit.clone( repositoryUrl, gitWorkingDirectoryPath );
+	await simpleGit.clone( repositoryUrl, gitWorkingDirectoryPath, [
+		'--depth=1',
+		'--no-single-branch',
+	] );
```

Comment:
Did you actually try running the tool after this change? last I tried a shallow close it broke?

Reply:
I ran the tool with everything except this step commented out and confirmed that, in the cloned repo, I could still checkout `release/8.4` and `origin/release/8.4`. Not sure how I can test the changes without releasing Gutenberg?

Do you remember what broke? `--no-single-branch` means that switching branches is still possible.

Reply:
yes, I definitely didn't know about `--no-single-branch` so it  might be ok, just needed  to confirm. I think we should do the same for the  perf  tests :)

Reply:
This change affects the `perf` command as well because `lib/git.js` is shared.

Reply:
oh right 👍

Reply:
That would be amazing to download less data during all type of releases ❤️